lib/brcobranca/boleto/bradesco.rb

```diff
@@ -46,7 +46,10 @@ def nosso_numero_boleto
       end
 
       def nosso_numero_dv
-        "#{carteira}#{numero_documento}".modulo11_bradesco
+        "#{carteira}#{numero_documento}".modulo11(
+          multiplicador: [2, 3, 4, 5, 6, 7],
+          mapeamento: {10 => 'P', 11 => 0}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.